api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -161,6 +161,10 @@ type AgentClusterInstallSpec struct {
 	// Proxy defines the proxy settings used for the install config
 	// +optional
 	Proxy *Proxy `json:"proxy,omitempty"`
+
+	// SchedulableMasters enables ability to schedule workload on masters
```

Comment:
It might be worth mentioning what would be the default value here.

Reply:
I thought it's as simple, but then I had a look at other fields... In swagger we define defaults (i.e. https://github.com/openshift/assisted-service/blob/master/swagger.yaml#L7063-L7066) but then here in _types.go we do not mention them, e.g.

```
	// DiskEncryption is the configuration to enable/disable disk encryption for cluster nodes.
	// +optional
	DiskEncryption *DiskEncryption `json:"diskEncryption,omitempty"`
```

So what is our convention? I'm happy do add a default for SchedulableMasters, but I'd like to be clear on the fact that it will be the first default mentioned in this struct.